consumer/hack/bind.sh

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env bash
+
+#quit if exit status of any cmd is a non-zero value
+set -euo pipefail
+
+SCRIPT_DIR="$(
+    cd "$(dirname "$0")" >/dev/null
+    pwd
+)"
+
+# shellcheck source=consumer/hack/run_test_workload.sh
+source "$SCRIPT_DIR/run_test_workload.sh"
+
+usage() {
+    echo "
+Usage:
+    ${0##*/} [options]
+
+Setup Pipeline Service on a kcp workspace.
+
+Mandatory arguments:
+    -f, --from SERVICE_WORKSPACE
+        Path to the workspace exposing the APIExport.
+
+Optional arguments:
+    -k, --kubeconfig KUBECONFIG
+        Path to the kubeconfig with the connection information to the kcp workspace.
+    -t, --to SERVICE_WORKSPACE
+        Workspace in which to bind the pipeline service.
+        Default: Current workspace defined in KUBECONFIG
+    -d, --debug
+        Activate tracing/debug mode.
+    -h, --help
+        Display this message.
+
+Example:
+    ${0##*/} --from root:pipeline-service --to '~:pipeline-service-test'
+" >&2
+}
+
+parse_args() {
+    while [[ $# -gt 0 ]]; do
+        case $1 in
+        -k | --kubeconfig)
+            shift
+            KUBECONFIG="$1"
+            ;;
+        -f | --from)
+            shift
+            SERVICE_WORKSPACE="$1"
+            ;;
+        -to | --to)
+            shift
+            WORKSPACE="$1"
```

Comment:
Suggestion: Change it to USER_WORKSPACE to clearly differentiate between service and user's workspaces.

Reply:
Done